### PR TITLE
update ChannelUpdate message to not use excessdata as an optional field

### PR DESCRIPTION
**Pull request by joemphilips**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## Overview
+
+This repository consists from a solution which contains four main packages
```
Comment (knocte) [inline README.md]: typo: consists from -> consists of (I think)

File: README.md [hunk 2]
```diff
@@ -6,22 +13,19 @@ That is, it will read in following order.
...
 * Read `appsettings.json`
 
-## Supported platforms
-
-* By default, the solution is built with dependencies on Secp256k1.Net and NSec (which have native dependencies that need to be built for every platform, and we right now only provide Linux binaries). This version is published as `DotNetLightning.Core` on Nuget.
-* For a more portable version (but possibly less performant) you can compile with BouncyCastle support, passing /p:BouncyCastle=True to your build. This version is published as `DotNetLightning` on Nuget.
+## Supported platforms for the package `DotNetLightning.Core`
 
-## TODO
+The package `DotNetLightning.Core` is published to nuget as two different style.
```
Comment (knocte) [inline README.md]: typo: style -> styles

Comment (knocte) [inline README.md]: typo: style -> styles
(BTW I would not use the name "DotNetLightning.Core" in this sentence, as nuget names are mentioned below

File: README.md [hunk 3]
```diff
@@ -6,22 +13,19 @@ That is, it will read in following order.
...
+* By default, the solution is built with dependencies on Secp256k1.Net and NSec (which have native dependencies that need to be built for every platform, and we right now only provide Linux/MacOS binaries). This version is published as `DotNetLightning.Core` on Nuget.
+* For a more portable version (but possibly less performant) you can compile with BouncyCastle support, passing /p:BouncyCastle=True to your build. This version is published as `DotNetLightning` on Nuget.
 
 ## Developer notes
 
 ![Alt text](images/Package_Dependency_Graph.png?raw=true "Package dependency graph")
 
 Purple background indicates that the package is included in this repository.
-Important point here is that we should have at least external dependencies which may become possible
+Important point here is that we should have at least external dependencies as possible to reduce number of possible
```
Comment (knocte) [inline README.md]: typo: at least -> the least
typo: attack vector -> attack vectors

File: README.md (hunk 1 above)
Comment (joemphilips) [reply README.md]: "This repository contains one solution which ..." might be better.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+## Overview
+
+This repository contains one solution which contains four main packages
```
Comment (knocte) [inline README.md]: I would use the word "project" here, not "package" (to not confuse it with nuget terminology)

File: README.md [hunk 5]
```diff
@@ -8,21 +15,18 @@ That is, it will read in following order.
 
 ## Supported platforms
 
-* By default, the solution is built with dependencies on Secp256k1.Net and NSec (which have native dependencies that need to be built for every platform, and we right now only provide Linux binaries). This version is published as `DotNetLightning.Core` on Nuget.
-* For a more portable version (but possibly less performant) you can compile with BouncyCastle support, passing /p:BouncyCastle=True to your build. This version is published as `DotNetLightning` on Nuget.
-
-## TODO
+The package `DotNetLightning.Core` is published to nuget as two different styles.
```
Comment (knocte) [inline README.md]: I would change this to `The project DotNetLightning.Core...`

File: README.md (hunk 4 above)
Comment (joemphilips) [reply README.md]: Good point. BTW you may try "suggestion" feature of github.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ## Overview
 
-This repository contains one solution which contains four main packages
+This repository contains one solution which contains four main project.
```
Comment (knocte) [inline README.md]: typo: project -> projects

**Pull request merged**